test/sequential/test-inspector-esm.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+const { NodeInstance } = require('../common/inspector-helper.js');
+
+function checkListResponse(response) {
+  assert.strictEqual(1, response.length);
```

Comment:
Small nit: reverse the argument order here.

Reply:
Done, here and below.